src/main/resources/resource/Arduino/generate/arduinoMsgs.schema

```diff
@@ -71,9 +71,14 @@
 < publishI2cData/deviceId/[] data
 
 # NeoPixel
-> neoPixelAttach/deviceId/pin/b32 numPixels/depth
-> neoPixelSetAnimation/deviceId/animation/red/green/blue/b16 speed
+> neoPixelAttach/deviceId/pin/b16 numPixels/depth
```

Comment:
why did we change to a 16bit here vs 32 from before?

Reply:
because no one will have 65,535 pixels, max do to processing speed is about 2K (very rare)

Reply:
adafruit library says "uint16_t n"   .. so it makes sense to make it 16 here. just noticed the difference.